### PR TITLE
BlueRidge: LED Config support

### PR DESCRIPTION
**Pull request by Alpana07**
Led configs files for BlueRidge system 2s2u, 2s4u, 1s4u

### CODE REVIEW

**Comment by PriyangaRamasamy** [comment]
1. Blueridge has 3 flavours of system, 2S2U, 2S4U and 1S4U. Can you create the missing one.
2. Can you update the configs/config_map.json with the blueridge json details.

**Comment by PriyangaRamasamy** [comment]
can you please update the PR heading with system name

**Comment by Alpana07** [comment]
> 1. Blueridge has 3 flavours of system, 2S2U, 2S4U and 1S4U. Can you create the missing one.
> 2. Can you update the configs/config_map.json with the blueridge json details.

Done

**Comment by Alpana07** [comment]
> can you please update the PR heading with system name

Done

**Review by SunnySrivastava1984 — APPROVED** [review]

File: configs/config_map.json [hunk 1]
```diff
@@ -6,6 +6,9 @@
         "Rainier 2U": "ibm,rainier-2u",
         "Rainier 2U Pass 1": "ibm,rainier-2u",
         "Rainier 4U": "ibm,rainier-4u",
-        "Rainier 4U Pass 1": "ibm,rainier-4u"
+        "Rainier 4U Pass 1": "ibm,rainier-4u",
+        "Blueridge 1S4U": "ibm,blueridge-1s4u",
+        "Blueridge 2S2U": "ibm,blueridge-2u",
+        "Blueridge 2S4U": "ibm,blueridge-4u"
```
Comment (jinuthomas) [inline configs/config_map.json:12]: Please move it above , as this is alphabetically arranged

**Review by jinuthomas — COMMENTED** [review]
just a small rearrangement , else everything is fine

**Pull request merged**